packages/Tabs/src/tabs.scss

```
@@ -1,115 +1,212 @@
-  --ids-tab-icon-premium-color: #{tokens.$dandelion-600};
-
-  /* === Inline Style === */
-
-  --ids-inline-tab-button-font-size: #{tokens.$font-size-3};
-
-  /* Tab */
-  --ids-inline-tab-button-padding: calc(#{tokens.$space-1} + #{tokens.$space-2}) #{tokens.$space-3};
-  --ids-inline-tab-button-after-bottom: -#{tokens.$space-1};
-  --ids-inline-tab-button-after-height: #{tokens.$space-1};
-  --ids-inline-tab-button-after-horizontal-space: #{tokens.$space-0};
+    /* Default */
+    --ids-tabs-font-family: #{tokens.$primary-font-family};
+    --ids-tabs-font-size: #{tokens.$font-size-4};
+    --ids-tabs-margin: #{tokens.$space-0} #{tokens.$space-0} 2rem;
+
+    /* Tab */
+    --ids-tab-button-padding: #{tokens.$space-2};
+    --ids-tab-button-font-size: #{tokens.$font-size-4};
+    --ids-tab-button-font-weight: #{tokens.$font-weight-medium};
+    --ids-tab-button-line-height: #{tokens.$line-height-xxxl};
+    --ids-tab-button-color: #{tokens.$grey-600};
+    --ids-tab-button-hover-color: #{tokens.$electric-blue-500};
+    --ids-tab-button-active-color: #{tokens.$electric-blue-500};
+    --ids-tab-button-after-horizontal-space: #{tokens.$space-2};
+    --ids-tab-button-after-height: calc(#{tokens.$space-1} / 2);
+    --ids-tab-button-after-bottom: calc(#{tokens.$space-2} - var(--ids-tab-button-after-height));
+    --ids-tab-button-after-background: #{tokens.$electric-blue-500};
+    --ids-tab-button-focus-outline-size: 0.2rem;
+    --ids-tab-button-focus-outline-color: var(--hop-primary-surface-strong);
+    --ids-tab-button-focus-outline-radius: var(--hop-shape-rounded-sm);
+    --ids-tab-button-focus-height: 100%;
+    --ids-tab-button-focus-width: 100%;
+
+    /* Icons */
+    --ids-tab-icon-margin-left: #{tokens.$space-1};
+    --ids-tab-icon-notification-color: #{tokens.$coral-500};
+    --ids-tab-icon-premium-color: #{tokens.$dandelion-600};
+
+    /* === Inline Style === */
+
+    --ids-inline-tab-background-color: transparent;
+    --ids-inline-tab-border-radius: 0;
+    --ids-inline-tab-outline: none;
+    --ids-inline-tab-outline-offset: 0;
+    --ids-inline-tab-button-font-size: #{tokens.$font-size-3};
+    --ids-inline-tab-button-line-height: #{tokens.$line-height-xxxl};
+    --ids-inline-tab-button-focus-height: 100%;
+    --ids-inline-tab-button-focus-width: 100%;
+
+    /* Tab */
+    --ids-inline-tab-button-padding: calc(#{tokens.$space-1} + #{tokens.$space-2}) #{tokens.$space-3};
+    --ids-inline-tab-button-after-bottom: -#{tokens.$space-1};
+    --ids-inline-tab-button-after-height: #{tokens.$space-1};
+    --ids-inline-tab-button-after-horizontal-space: #{tokens.$space-0};
 }
 
-.ids-tabs {
-  font-family: var(--ids-tabs-font-family);
-  font-size: var(--ids-tabs-font-size);
+[data-brand="workleap"] {
+    /* Default */
+    --ids-tabs-font-family: var(--hop-body-sm-font-family);
+    --ids-tabs-font-size: var(--hop-body-sm-font-size);
+    --ids-tabs-margin: #{tokens.$space-0} #{tokens.$space-0} 1.25rem;
```

Igloo tokens must be removed even if there are no equivalent tokens in 🐸Hopper.
```suggestion
    --ids-tabs-margin: 0 0 1.25rem;
```